assembly.xml

```diff
@@ -1,51 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<!--
- ~ Copyright 2018-present HiveMQ GmbH
- ~
- ~  Licensed under the Apache License, Version 2.0 (the "License");
- ~  you may not use this file except in compliance with the License.
- ~  You may obtain a copy of the License at
- ~
- ~        http://www.apache.org/licenses/LICENSE-2.0
- ~
- ~  Unless required by applicable law or agreed to in writing, software
- ~  distributed under the License is distributed on an "AS IS" BASIS,
- ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- ~  See the License for the specific language governing permissions and
- ~  limitations under the License.
- -->
-
-<assembly xmlns="http://maven.apache.org/xsd/assembly-1.1.2.xsd"
-          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-          xsi:schemaLocation="http://maven.apache.org/xsd/assembly-1.1.2.xsd">
-    <id>distribution</id>
-    <formats>
-        <format>zip</format>
-    </formats>
-    <files>
-        <file>
-            <source>target/LICENSE.txt</source>
```

Comment:
The license and readme are now not part of the zip anymore, is this intended?
If not, we should include them as in: https://github.com/hivemq/hivemq-allow-all-extension/blob/master/build.gradle.kts